wfl/Controllers/MenuSelectionsController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Web.Http;
+using System.Threading.Tasks;
+using System.Linq;
+using System.Collections.Generic;
+using Symphono.Wfl.Models;
+using Symphono.Wfl.Database;
+
+namespace Symphono.Wfl.Controllers
+{
+    public class MenuSelectionsController : ApiController
+    {
+        private IDBManager dbManager { get; }
+        public MenuSelectionsController(IDBManager dbManager) {
+           this.dbManager = dbManager;
+        }
+
+        [Route("api/food-order/{foodOrderId}/menu-selection")]
+        [HttpGet]
+        public async Task<IHttpActionResult> GetAsync([FromUri] string foodOrderId)
+        {
+            FoodOrder order = await dbManager.GetEntityByIdAsync<FoodOrder>(foodOrderId);
+            return Ok(order.MenuSelections);
+        }
+
+        [Route("api/menu-selection")]
```

Comment:
This is a question about path names in general. Paths that I would expect to be plural are singular in many places. What I mean is that this GET request (and many others throughout the API) returns many menu-selections. This might just be a topic to discuss in IPM next week, but it is a bit confusing to me.